fontforgeexe/fontinfo.h

```diff
@@ -0,0 +1,21 @@
+#ifndef FONTFORGE_FONTFORGEEXE_FONTINFO_H
+#define FONTFORGE_FONTFORGEEXE_FONTINFO_H
+
+#include <ggadget.h>
```

Comment:
@frank-trampe hmm I'm not sure about that. I always thought that quotes vs angle brackets was more about if it was quotes, it would search the current folder before falling back to whatever search paths were specified. [This](https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html) certainly seems to support that idea.